ibis/__init__.py

```diff
@@ -12,18 +15,29 @@
 from ibis.expr import api
 from ibis.expr.api import *  # noqa: F401,F403
 
-try:
-    import importlib.metadata as importlib_metadata
-except ImportError:
-    # TODO: remove this when Python 3.9 support is dropped
-    import importlib_metadata
-
-__all__ = ['api', 'ir', 'util', 'IbisError', 'options']
+__all__ = ['api', 'ir', 'util', 'BaseBackend', 'IbisError', 'options']
```

Comment:
I'm not clear on why we're exposing `BaseBackend` here

Reply:
:shrug:. It was the only thing exposed in the top level namespace that wasn't also part of `__all__`. Figured we might as well be consistent here.